scripts/deps.sh

```diff
@@ -22,10 +22,14 @@
 ctrl_c()
 {
   exit $?
-} 
+}
 trap ctrl_c SIGINT
 
 # First load snap deps
+echo "Installing dependencies if not exists"
+go get github.com/tools/godep
+go get github.com/intelsdi-x/snap/control
```

Comment:
@conikeec: An alternative  might be to error out with a message that would help someone know to add snap to their GOPATH.

@intelsdi-x/snap-maintainers: Other thoughts?